app/Models/NetworkData/Atc.php

```diff
@@ -187,6 +187,7 @@ public static function scopeIsUK($query)
                 ->orWhere('callsign', 'LIKE', "STC\_%")
                 ->orWhere('callsign', 'LIKE', "LON\_%")
                 ->orWhere('callsign', 'LIKE', "LTC\_%")
+                ->orWhere('callsign', 'LIKE', 'MAN\_%')
                 ->orWhere('callsign', 'LIKE', 'EGGX%')
                 ->orWhere('callsign', 'LIKE', 'EGTT%')
                 ->orWhere('callsign', 'LIKE', 'EGPX%');
```

Comment:
```suggestion
                ->orWhere('callsign', 'LIKE', 'EGPX%');
                ->orWhere('callsign', 'LIKE', 'LXGB_%');
                ->orWhere('callsign', 'LIKE', 'LCRA_%');
                ->orWhere('callsign', 'LIKE', 'FHAW_%');
```

Reply:
I noticed when I did the manual queries that we were missing some of the minor positions. Might as well add them whilst we are at it!